ficsit-companion/src/main.cpp

```diff
@@ -113,8 +113,9 @@ bool Render(SDL_Window* window, App* app)
 
     ImGui::SetNextWindowPos(ImVec2(0, 0));
     ImGui::SetNextWindowSize(ImGui::GetIO().DisplaySize);
-
+    
```

Comment:
Trailing whitespaces should be removed

Reply:
ahh thought it would be to seperate, cause in 113 theres also a whitespace

Reply:
There is a new line, but not a new line with "useless" spaces

113 is "\n"
116 is "    \n"

Reply:
Ah now i get it, how du i commit my change in the pull request?
or do i make a new one

Reply:
I think if you push the commit to your branch it will be added to the PR

Reply:
fixed, thanks commited to wrong branch xD